test/WebJobs.Script.Tests.Integration/ScriptHostEndToEnd/NodeScriptHostTests.cs

```diff
@@ -55,17 +55,17 @@ public NodeScriptHostTests(TestFixture fixture)
 
             var result = (IActionResult)request.HttpContext.Items[ScriptConstants.AzureFunctionsHttpResponseKey];
 
-            Assert.IsType<ScriptObjectResult>(result);
+            Assert.IsType<RawScriptResult>(result);
```

Comment:
I think that the tests in NodeContentTests should be thorough enough to cover enableContentNegotiation=true, since that requires the triggered function to provide that flag as a result